src/instances/types/Array/reconcile.ts

```diff
@@ -32,7 +32,7 @@ export default function (newAbstractElements: PlusnewAbstractElement[], instance
       if (!found) {
```

Comment:
für bessere Lesbarkeit bitte hier "(found === false)" verwenden .

Reply:
> für bessere Lesbarkeit bitte hier "(found === false)" verwenden .
agreed

please switch to english in this public repository